frontend/public/components/namespace.jsx

```diff
@@ -847,13 +852,37 @@ export const ProjectsPage = (props) => {
   const isPrometheusAvailable = usePrometheusGate();
   const showMetrics = isPrometheusAvailable && canGetNS && window.screen.width >= 1200;
   const showActions = showMetrics;
+  const onCreate = React.useCallback((pluginUID) => createProjectModal({ pluginUID }), [
+    createProjectModal,
+  ]);
+  const createProps = React.useMemo(
+    () =>
+      createProjectModalExtensions.length > 1
+        ? {
+            action: onCreate,
+            items: resolved
+              ? createProjectModalExtensions.reduce(
+                  (acc, { uid, properties, pluginName }) => ({
+                    ...acc,
+                    [uid]: properties.label || t('Create {{pluginName}} Project', { pluginName }),
```

Comment:
I might adjust the fallback label here. It sounds like it's a button to create a project _for_ the plugin.

Maybe "Create Project with {pluginName} options"? Better might be to use the plugin display name if we have it.

Reply:
What about just "Crate Project using {{pluginName}}"